### PR TITLE
Design document 

### PR DESCRIPTION
**Pull request by aniagaska**
I've added the typefaces and colors to the readme for easier access!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,44 @@
...
+
+[DM Mono](https://fonts.google.com/specimen/DM+Mono)
+
+## Headlines
+
+### H1, H2
+[Fjalla One](https://fonts.google.com/specimen/Fjalla+One) 
+
+```
+
```
Comment (bronzehedwick) [inline README.md]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,44 @@
...
+Text-transform: uppercase; 
+Font-weight: bold;
+
```
Comment (bronzehedwick) [inline README.md]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,44 @@
...
+## Primary Typeface
+
+[DM Mono](https://fonts.google.com/specimen/DM+Mono)
+
+## Headlines
+
+### H1, H2
+[Fjalla One](https://fonts.google.com/specimen/Fjalla+One) 
+
+```
```
Comment (bronzehedwick) [inline README.md]:
```suggestion
```css
```

File: README.md [hunk 4]
```diff
@@ -1,2 +1,44 @@
...
+
+Text-transform: uppercase; 
+Font-weight: bold;
+
+```
+### H3, H4...
+[DM Mono](https://fonts.google.com/specimen/DM+Mono)
+
+```
```
Comment (bronzehedwick) [inline README.md]:
```suggestion
```css
```

File: README.md [hunk 5]
```diff
@@ -1,2 +1,44 @@
...
+
```
Comment (bronzehedwick) [inline README.md]:
```suggestion
```

File: README.md [hunk 6]
```diff
@@ -1,2 +1,44 @@
...
+Font-family: DM Mono;
+Font-weight: 500;
+
```
Comment (bronzehedwick) [inline README.md]:
```suggestion
```

File: README.md [hunk 7]
```diff
@@ -1,2 +1,44 @@
...
+```
+
+## Colors
+
+### Primary Color
+
+Yellow #FFDA00
```
Comment (bronzehedwick) [inline README.md]:
```suggestion
Yellow `#FFDA00`
```

File: README.md [hunk 8]
```diff
@@ -1,2 +1,44 @@
...
+
+### Secondary Colors
+
+*Donatello Color Palette*
+
+* Bright Purple #B947A7
```
Comment (bronzehedwick) [inline README.md]:
```suggestion
* Bright Purple `#B947A7`
```

File: README.md [hunk 9]
```diff
@@ -1,2 +1,44 @@
...
+* Dark Purple #944CAD
```
Comment (bronzehedwick) [inline README.md]:
```suggestion
* Dark Purple `#944CAD`
```

File: README.md [hunk 10]
```diff
@@ -1,2 +1,44 @@
...
+* Teal #49B775
```
Comment (bronzehedwick) [inline README.md]:
```suggestion
* Teal `#49B775`
```

File: README.md [hunk 11]
```diff
@@ -1,2 +1,44 @@
...
+* Green #98BB35
```
Comment (bronzehedwick) [inline README.md]:
```suggestion
* Green `#98BB35`
```

**Review by bronzehedwick — APPROVED** [review]
Looks good, thanks!

I made some markdown suggestions which you can either accept or reject from the github interface. Approving either way.

File: README.md (hunk 1 above)
Comment (aniagaska) [reply README.md]: Thank you Chris! I've applied the changes.

**Pull request merged**